### DISCUSSION TITLE
pip install

### DISCUSSION THREAD

**Discussion by wade666000** [DavidMGilbert/btc-hack#56] [Q&A]
do you have to install the three pip installments with the new download version btc-hackv2?

**Comment by farshadjanu1** [comment]
> do you have to install the three pip installments with the new download version btc-hackv2?
> 

yes

**Reply by MvdS74** [reply]
But the script doesn't work, even with the requirements installed. So don't bother...

Most of the generated mnemonic phrases are invalid (No, it's not just random selecting words. There are rules). The code to create the key from the mnemonic doesn't work. (All wallet addresses need to be in a specific format, and the keys generated from this script are all invalid) And the API used blocks you after 1 or 2 seconds because it has very strict limits. But because there is no error handling, the script doesn't tell you that, it suggests that it is checking wallets, but it is not because the API has already blocked you after the 3rd call.
